dinesykmeldte-backend/src/main/kotlin/no/nav/syfo/minesykmeldte/model/PreviewSykmeldt.kt

```diff
@@ -22,11 +22,13 @@ data class PreviewSykmeldt(
 data class Oppfolgingsplan(
     val hendelseId: UUID,
     var tekst: String,
+    var mottatt: OffsetDateTime,
```

Comment:
Disse to må kanskje være optional? Altså, sånn at det ikke feiler hvis vi allerede har denne typen hendelser som ikke har noe `mottatt` :)

Reply:
Ja det er sant, skal endre på det!